src/main/java/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.java

```diff
@@ -2301,7 +2301,7 @@ public DefaultShardManagerBuilder setMaxBufferSize(int bufferSize)
      *         to whether or not loading has finished when this returns.
      */
     @Nonnull
-    public ShardManager build() throws LoginException, IllegalArgumentException
+    public DefaultShardManager build() throws LoginException, IllegalArgumentException
```

Comment:
Is this a breaking change?

Reply:
Yes a changed return type is breaking the ABI

Reply:
Would it better to add `login()` to the ShardManager interface?

Reply:
Yeah, I think that would be better.